aips/aip-81.md

```diff
@@ -173,20 +153,20 @@ The pepper request should be rejected if any check below fails.
 
 ### Pepper processing
 - Let `uid_val` be the user ID from `jwt.payload` indicated by `uid_key`.
-- Let `aud` be the effective client ID determined by `jwt.payload.aud` and `aud_override`.
-- Let `input` be a canonical serialization of data items `jwt.payload.iss, uid_key, uid_val, aud`.
-- Compute `pepper` as `vuf.eval(vuf_sk, input)`.
-- Let `pepper_bytes` be a serialization of the VUF output `pepper`.
-- Compute `pepper_encryped` as `asymmetric.enc(epk, pepper_bytes)`.
-
-### Pepper response (the output)
-
-The pepper response from the pepper service to the user should simply be the pepper encrypted by the EPK, hexlified, and wrapped in a JSON.
-
-Below is an example pepper response, assuming `vuf=Bls12381G1Bls` and `asymmetric=ElGamalCurve25519Aes256Gcm`.
+- Compute `pepper_input` to be a canonical serialization of data items `(jwt.payload.iss, uid_key, uid_val, jwt.payload.aud)`.
```

Comment:
I think we should explain the actual serialization format we use in this AIP.

Same for the signature/VUF scheme.

This PR, we can push as is. But maybe let's aim to make it more "concrete." Since it will be helpful for us & the community to know the exact formats.

Reply:
fixed